.github/workflows/build_and_deploy.yml

```diff
@@ -8,8 +8,8 @@ on:
 env:
   NAPI_CLI_VERSION: 2.16.2
   TURBO_VERSION: 1.10.9
-  NODE_MAINTENANCE_VERSION: 16
-  NODE_LTS_VERSION: 18
+  NODE_MAINTENANCE_VERSION: 18.18.2
```

Comment:
Can we test agains 18 so it automatically tests minor and patch releases instead of pinning?

Reply:
I did so first, but changed in https://github.com/vercel/next.js/pull/56896/commits/6c00b6942b8836a803aed29970a85534b14f010c as <18.18.2 was being installed, which caused issues.

See https://github.com/vercel/next.js/actions/runs/6536411624/job/17748103891?pr=56896#step:28:94

Reply:
Actually, that value seems to be unused, it was just being copied over from other similar workflows.